src/components/RateAgentModal/RateAgentTags.tsx

```diff
@@ -18,6 +20,7 @@ export default function RateAgentTags({ control }: IRateAgentTags) {
       render={({ field }) => {
         const { value, onChange } = field
 
+        //TODO: Assure form value compatibility with translations
```

Comment:
I'm out of practice with react-hook-form so I'd want to see how the field values interact with the forms. The forms should always send through the English form values but display the translated copy. Right now I've left it as-is to handle in another PR.

Reply:
Good catch. The translationMap sets this up well, we just need to make sure the value of that tag as used in handleClick is the key, and the displayed text is the translation. I'll open a PR after this one goes in.